ecosystem/sep-0006.md

```diff
@@ -317,7 +375,7 @@ Request parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | Code of the asset the user wants to withdraw. The value passed must match one of the codes listed in the [/info](#info) response's withdraw object.
+`asset_code` | string | (optional) Code of the asset the user wants to withdraw. The value passed must match one of the codes listed in the [/info](#info) response's withdraw object. This field is mandatory if `quote_id` is not set.
```

Comment:
I think here the `asset_code` and `destination_asset` definitions make sense. I think `asset_code` should be mandatory though – why change that?

Also, I'm a bit confused about the overall developer / user experience. What should the anchor provide on `/info`? Is it:

a) the list of on-chain assets that can be withdrawn, or 
b) the list of off-chain assets they convert to?

Intuitively I'd go with (a) because of the current behavior. Is that what you're thinking?

Reply:
> I think here the `asset_code` and `destination_asset` definitions make sense.

Is SEP-6 `asset_code` always the code of the asset in the Stellar network?

Reply:
> Also, I'm a bit confused about the overall developer / user experience. What should the anchor provide on `/info`? Is it:
> 
> a) the list of on-chain assets that can be withdrawn, or
> b) the list of off-chain assets they convert to?
> 
> Intuitively I'd go with (a) because of the current behavior. Is that what you're thinking?

We should stick to the current behavior and avoid any breaking changes.

Reply:
> Is SEP-6 asset_code always the code of the asset in the Stellar network?

See https://github.com/stellar/stellar-protocol/pull/973#discussion_r651919114 – I don't think that's true as of yet (it's ambiguous), but that's what I'm proposing we standardize for.